spec/lucky/action_callbacks_spec.cr

```diff
@@ -207,7 +207,7 @@ private def with_log
     log_formatter: RawLogFormatter
   )
 
-  Lucky.temp_config(logger: logger) do
+  Lucky.temp_config(logger: logger, action_pipe_log_level: Logger::Severity::INFO) do
```

Comment:
Actually, should we just call these `before/after` macros callbacks? We kind of have callbacks and pipes in the code and guides so I think we should choose one.

I partly named them pipes because they work a bit more similarly to pipes in Elixir where you have to return an explicit response/continue. And also because "callbacks" is a bad word to some people. But maybe we should just call them callbacks since that may make more sense. Thoughts? @edwardloveall @jwoertink

Reply:
I think pipes is fine, but I don't think I feel too strongly either way. There's also an ick factor about callbacks for me, but not so strong that I'd steer away from the word itself if it's the best word to discribe.

Reply:
I like pipes because it sounds cool. Maybe that's because of callbacks having a bad taste, but we use the term "pipe" in our apps. I would like to see it a bit more standardized, and consistent though. 

Like we have:
```
module RequireSitePipe
  macro included
    before require_site
  end

  private def require_site
    if current_site?
      continue
    else
      raise Lucky::RouteNotFoundError.new(context)
    end
  end

  private def current_site : SiteHelpers::Site
    current_site?.not_nil!
  end
end
```

but just by looking at the name of the module, it doesn't immediately tell you why we added "Pipe" to it. We could have just called it `RequireSite`.

Reply:
As mentioned in our gitter room I'll stick with pipes. I was just worried it was confusing. I'll make some changes to the code to be pipe instead of callback!